scripts/install_avalanchego_release.sh

```diff
@@ -11,41 +11,119 @@ source "$SUBNET_EVM_PATH"/scripts/versions.sh
 # Load the constants
 source "$SUBNET_EVM_PATH"/scripts/constants.sh
 
-VERSION=$AVALANCHEGO_VERSION
-
 ############################
 # download avalanchego
 # https://github.com/ava-labs/avalanchego/releases
 GOARCH=$(go env GOARCH)
 GOOS=$(go env GOOS)
-BASEDIR=${BASE_DIR:-"/tmp/avalanchego-release"}
+BASEDIR=${BASEDIR:-"/tmp/avalanchego-release"}
+
 mkdir -p ${BASEDIR}
+
+VERSION=$AVALANCHEGO_VERSION
```

Comment:
Would it be better to use a single variable for this instead of `VERSION` and `AVALANCHEGO_VERSION` ?

Reply:
Yeah, I didn't really mean to move this. I kept `VERSION` because it makes the next couple of lines shorter. Happy to change it though